app/src/main/java/ru/vssemikoz/newsfeed/data/NewsStorage.java

```diff
@@ -2,20 +2,20 @@
 
 import java.util.List;
 
-import ru.vssemikoz.newsfeed.MainApplication;
 import ru.vssemikoz.newsfeed.dao.NewsItemDAO;
+import ru.vssemikoz.newsfeed.database.NewsAppDataBase;
 import ru.vssemikoz.newsfeed.models.Category;
 import ru.vssemikoz.newsfeed.models.NewsItem;
 
-public class NewsStorage {
-
+public class NewsStorage implements DataStorage {
```

Comment:
I think we should have `NewsStorage` interface and `LocalNewsStorage` implementation.